examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Examples
+
+## Overview
+
+This directory contains example applications that showcase how to use RIOT-rs.
+
+- [benchmark/](./benchmark): how to use `benchmark()`
+- [embassy-net-udp/](./embassy-net-udp): UDP echo example
+- [embassy-net-tcp/](./embassy-net-tcp): TCP echo example
+- [embassy-http-server/](./embassy-http-server): HTTP server example
+- [embassy-usb-keyboard/](./embassy-usb-keyboard): USB HID example
+- [hello-world/](./hello-world): a classic
+- [minimal/](./minimal): minimized to the max RIOT-rs config
+- [threading/](./threading): how to start and use preemptively scheduled threads
+
+## Networking
+
+Some examples configure networking. By default, they will listen on a static
+IPv4 address. Be sure to configure the host computer end of the network
+accordingly.
+
+The default IPv4 address the test examples will configure is `10.42.0.61`.
+You can configure the other end like this:
+
+    $ ip address add 10.42.0.1/24 dev <DEVICE>
```

Comment:
ip addr ?

Reply:
'address' is actually the normal subcommand name. do you mean we should use a shorter version? The shortest would be just `ip a a ...`.

Reply:
OK, let's keep it like that ;)